README.md

```diff
@@ -1,49 +1,48 @@
 # Terraform GSG on GCP with Cloud Shell
 
-A Getting Started Guide for Terraform and Google Cloud Platform, using Google's interactive [Cloud Shell](https://cloud.google.com/shell/ "Google Cloud Shell homepage").
+A Getting Started Tutorial for Terraform and Google Cloud Platform, using
+Google's interactive [Cloud Shell](https://cloud.google.com/shell/).
 
 ## Purpose
 
-This guide will help you learn how to use [Terraform](https://www.terraform.io/intro/index.html "Introduction to Terraform"), an open source "Infrastructure as Code" tool provided by Hashicorp.
+This tutorial will help you learn how to use
+[Terraform](https://www.terraform.io/intro/index.html "Introduction to
+Terraform"), an open source "Infrastructure as Code" tool provided by HashiCorp.
```

Comment:
I would combine these two sections together since they both "introduce" the purpose of this repo, then create another section for prerequisites